src/app/containers/Metadata/linkedDataAbout.js

```diff
@@ -0,0 +1,39 @@
+const acceptableTypes = ['Person', 'Event', 'Organization', 'Place'];
+
+export const checkType = types => {
+  if (types.length === 0 || types.length > 1) {
+    return 'Thing';
+  }
+  return acceptableTypes.includes(types[0]) ? types[0] : 'Thing';
+};
+
+export const checkSameAs = uris => {
+  const sameAs = uris.filter(uri => uri.includes('http://dbpedia.org'));
+  return sameAs.length ? sameAs : undefined;
```

Comment:
What are we trying to achieve with `sameAs.length` in this context? That `sameAs` is an array or similar? Also should we be returning `undefined`, would `false` or `null` be more appropriate for the function this returns to?

Reply:
Here the sameAs.length acts as a check, such that when there are no URIs from the http://dbpedia.org domain, the sameAs property will be set to undefined and not show up in the about object as an empty array

When set to false or null, the sameAs property will appear with a value of null or false in the about object